package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "rai-sdk-javascript",
+  "name": "@relationalai/rai-sdk-javascript",
   "description": "RelationalAI SDK for JavaScript",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
I think Brad started at 0.1.0 and then has been using `minor` to indicate breaking changes. I think we should do the same.